### ISSUE TITLE
Any command return help

### ISSUE THREAD

**Issue by AdrianoCahete** [jiahaog/nativefier#10]
Tried with Powershell and CMD (as administrator).

All commands using nativefier (include Mssenger example) returns a help info and nothing is created. I'm using a Win10 x64.

```
F:\Projetos\XXXX\xxxxxx\app\xxxxxxxApp>nativefier Messenger --target=http://messenger.com --platform=darwin --arch=x64 --versio
n=0.28.2
Usage: nativefier <appname> <target> --platform=<platform> --arch=<arch> --version=<version>

Required options

appname            name for the app
target             target url for the single page app
platform           all, or one or more of: linux, win32, darwin (comma-delimited if multiple)
(...)
```
```
F:\>node -v
v5.2.0

F:\>npm -v
3.3.12
```

**Comment by jiahaog** [comment]
Sorry for the late reply, do you still have this problem?

**Comment by somebody1234** [comment]
It appears that @AdrianoCahete used the wrong syntax. It should be:
`nativefier Messenger http://messenger.com --platform=darwin --arch=x64 --versio
n=0.28.2`
instead of:
`nativefier Messenger --target=http://messenger.com --platform=darwin --arch=x64 --versio
n=0.28.2`.

**Comment by AdrianoCahete** [comment]
I get syntax from readme (https://github.com/jiahaog/nativefier#usage).

So, readme needs to be updated?

**Comment by jiahaog** [comment]
@somebody1234 Alright, the cli usage was recently changed with big under the hood changes so that would have probably fixed your error. Could you try again after updating the tool? I've made some changes that should support authentication.

**Comment by AdrianoCahete** [comment]
I tried, but self cert still returns a blank page, like #69 . See you fixed in next release, so i'll try again when you update it.

**Comment by jiahaog** [comment]
It's fixed in the 6.6.2!
